### ISSUE TITLE
net-snmp-tools Not Installed -- check_snmp Not Working

### ISSUE THREAD

**Issue by rehashedsalt** [manios/docker-nagios#31]
The Alpine package `net-snmp-tools` provides the following utilities that Nagios needs for its `check_snmp` command to function:

* `snmpget`
* `snmpwalk`

Without them, checks with SNMP commands error out with "External command error with no output (return code: 3)". The solution should be as simple as adding the package to the Dockerfile.

**Comment by manios** [comment]
Hi @rehashedsalt ,

Thanks for the finding! I have reproduced the error by running:

```bash
/opt/nagios/libexec # ./check_snmp -H localhost -o dad
External command error with no output (return code: 3)
/opt/nagios/libexec # 
```

After installing `net-snmp-tools` package, it works:

```bash
/opt/nagios/libexec # ./check_snmp -H localhost -o dad
CRITICAL - Plugin timed out while executing system call
```

I will apply a fix and get back to you once it is ready.

Thanks!
Christos

**Comment by manios** [comment]
Hi @rehashedsalt !

I have released a new `latest` tag based on `alpine:3.14` with the appropriate changes. Can you please give it a try

```shell
docker pull manios/nagios:latest
# or
docker pull manios/nagios:4.4.6
```

and provide me with feedback ? I am closing the issue. If you find any discrepancy please post a comment here and we will take a look.

Thanks,
Christos

**Comment by rehashedsalt** [comment]
Initial inspection looks good. Thanks for the prompt fix!
